orderbook/src/main.rs

```diff
@@ -41,10 +41,12 @@ async fn main() {
     let chain_id = web3.eth().chain_id().await.expect("Could not get chainId");
     let domain_separator =
         DomainSeparator::get_domain_separator(chain_id.as_u64(), settlement_contract.address());
-
     let orderbook = Arc::new(OrderBook::new(domain_separator));
-    let filter = api::handle_all_routes(orderbook.clone())
-        .map(|reply| warp::reply::with_header(reply, "Access-Control-Allow-Origin", "*"));
+    let cors = warp::cors()
+        .allow_any_origin()
+        .allow_methods(vec!["GET", "POST", "DELETE", "OPTIONS"])
```

Comment:
since u are adding the list, I would just in case add PUT and PATCH, since we might allow in the future endpoints using those